specification/app/resource-manager/Microsoft.App/preview/2023-11-02-preview/examples/DotNetComponents_List.json

```diff
@@ -29,7 +29,7 @@
             "name": "yellowcat",
             "type": "Microsoft.App/managedEnvironments/dotNetComponents",
             "properties": {
-              "componentType": "AspireResourceServerApi",
+              "componentType": "MyOtherDotNetComponentType",
```

Comment:
Also, have separate PR to main this api-version

Reply:
Here is the PR: https://github.com/Azure/azure-rest-api-specs/pull/28096/files